lib/src/main/kotlin/com/swmansion/starknet/data/types/Responses.kt

```diff
@@ -161,15 +161,19 @@ data class SyncingResponse(
 ) : Syncing()
 
 @Serializable
-sealed class GetBlockWithTransactionsResponse {
-    abstract val transactions: List<Transaction>
+sealed class GetBlockWithResponse {
```

Comment:
Should we rename it to `GetBlockWithResponseBase` or something else?

Reply:
Continued there: https://github.com/software-mansion/starknet-jvm/pull/409#discussion_r1507605842